includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1150,7 +1150,9 @@ public function update_existing_intent( $intent, $order, $prepared_source ) {
 			return $intent;
 		}
 
-		$level3_data = $this->get_level3_data_from_order( $order, get_option( 'woocommerce_store_postcode' ) );
+		$level3_data = WC_Stripe_Helper::is_wc_lt( '3.0' )
+			? array()
+			: $this->get_level3_data_from_order( $order, get_option( 'woocommerce_store_postcode' ) );
```

Comment:
Why are you individually checking for pre-3.0 on every `get_level3_data_from_order` call? Seems like you can move this logic within the method and only have it once.

Reply:
Yeh, I thought it was better to be explicit here but I can see that it's noisy. Moved the check to `get_level3_data_from_order` 👌